devel/cmark-gfm/Portfile

```diff
@@ -8,14 +8,18 @@ PortGroup           github 1.0
 # because the full version number is in the library's install name.
 # See https://github.com/commonmark/cmark/issues/474
 github.setup        github cmark-gfm 0.29.0.gfm.13
-revision            0
+github.tarball_from archive
 
-checksums           rmd160  4d46836c6a151360b0feac74c3f4c5b07a4ec2d1 \
-                    sha256  a95a166edd854f8c5206229d3848639eba438e0edba499b0575a9075545a7a30 \
-                    size    300099
+revision            1
```

Comment:
No need to bump the revision since nobody will have been able to install the port.

Reply:
removed